scripts/image

```diff
@@ -1,21 +1,22 @@
 #!/bin/bash
 
 ################################################################################
-#      This file is part of OpenELEC - http://www.openelec.tv
+#      This file is part of LibreELEC - https://libreelec.tv
+#      Copyright (C) 2016-present Team LibreELEC
```

Comment:
This should be "2018 Team LibreELEC" and in the future if we make further changes it can be changed to "2018-2020 Team LibreELEC" or whatever the date range is, or we continue in the future with the widely used but wrong "2018-present" marking. Claiming 2016-present now is wrong because we didn't do anything copyrightable to the file in 2016 or 2017.